doc/authentication.md

```diff
@@ -4,6 +4,7 @@ relatedlinks: https://www.youtube.com/watch?v=6O0q3rSWr8A
 ---
 
 (authentication)=
+
```

Comment:
I'm not a fan of these blank lines - are they required?
The reason I'm skeptical is that this is MyST syntax (not Markdown) and that it is giving an ID/anchor for the following heading. Adding a blank line inbetween makes that much less visible.

Reply:
Should we revert such changes? Even though these are Markdown files, but using MyST syntax.

Reply:
Is there a way to tweak the tool to make a difference between IDs and other content?
Because overall, I think we should require the blank lines - just not for IDs.